include/mender-inventory.h

```diff
@@ -62,13 +62,28 @@ mender_err_t mender_inventory_activate(void);
  */
 mender_err_t mender_inventory_deactivate(void);
 
+/**
+ * @brief Get mender inventory
+ * @param configuration Mender inventory key/value pairs table, ends with a NULL/NULL element, NULL if not defined
```

Comment:
@param name is "inventory" not configuration, you probably copy/paste which is good to keep the same presentation of the code but need to fix here ;)

Reply:
Fixed